src/p2p/net_node.inl

```diff
@@ -2483,26 +2484,38 @@ namespace nodetool
     std::vector<peerlist_entry> local_peerlist_new;
     zone.m_peerlist.get_peerlist_head(local_peerlist_new, true, max_peerlist_size);
 
+    /* Tor/I2P nodes receiving connections via forwarding (from tor/i2p daemon)
+    do not know the address of the connecting peer. This is relayed to them,
+    iff the node has setup an inbound hidden service. The other peer will have
```

Comment:
small typo here

Reply:
I don't see the typo, can you elaborate?

Reply:
I thought `iff` is supposed to be `if` but seems it's an abbreviation I wasn't aware of.